[2021-05-09 18:00:35,069] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-08T00:00:00+00:00 [queued]>
[2021-05-09 18:00:35,096] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dag_4.python_push_into_target 2021-05-08T00:00:00+00:00 [queued]>
[2021-05-09 18:00:35,096] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:00:35,097] {taskinstance.py:1069} INFO - Starting attempt 1 of 3
[2021-05-09 18:00:35,097] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2021-05-09 18:00:35,225] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): python_push_into_target> on 2021-05-08T00:00:00+00:00
[2021-05-09 18:00:35,228] {standard_task_runner.py:52} INFO - Started process 6126 to run task
[2021-05-09 18:00:35,232] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dag_4', 'python_push_into_target', '2021-05-08T00:00:00+00:00', '--job-id', '118', '--pool', 'default_pool', '--raw', '--subdir', '/home/navneetsajwan/airflow/dags/dag__4.py', '--cfg-path', '/tmp/tmpz06gzu4o', '--error-file', '/tmp/tmpxbda6twv']
[2021-05-09 18:00:35,233] {standard_task_runner.py:77} INFO - Job 118: Subtask python_push_into_target
[2021-05-09 18:00:35,344] {logging_mixin.py:104} INFO - Running <TaskInstance: dag_4.python_push_into_target 2021-05-08T00:00:00+00:00 [running]> on host navneetsajwan-Lenovo-G50-80
[2021-05-09 18:00:35,484] {taskinstance.py:1281} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=navneet@metaorigins.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dag_4
AIRFLOW_CTX_TASK_ID=python_push_into_target
AIRFLOW_CTX_EXECUTION_DATE=2021-05-08T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-05-08T00:00:00+00:00
[2021-05-09 18:00:41,882] {logging_mixin.py:104} INFO - push_test
[2021-05-09 18:00:41,882] {python.py:118} INFO - Done. Returned value was: None
[2021-05-09 18:00:41,887] {taskinstance.py:1185} INFO - Marking task as SUCCESS. dag_id=dag_4, task_id=python_push_into_target, execution_date=20210508T000000, start_date=20210509T123035, end_date=20210509T123041
[2021-05-09 18:00:42,693] {taskinstance.py:1246} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-05-09 18:00:42,703] {local_task_job.py:146} INFO - Task exited with return code 0
